components/ui/organisms/ComingSoon.jsx

```diff
@@ -73,7 +79,7 @@ export default function ComingSoon() {
       <section className="p-5 lg:py-12 lg:px-32">
         <div className="max-w-7xl mx-auto">
           {/* Section Title */}
-          <div className="text-center font-semibold text-2xl lg:text-4xl text-[#00A65B] mb-10">
+          <div className="text-center font-semibold text-2xl lg:text-4xl text-[#00A65B] mb-10 ">
```

Comment:
There's a space after `mb-10`

Running Prettier should have fixed that. Did you run `yarn format`?

If not, please do so after making all the necessary changes to your code and before committing